docs/component-library/existing/Admonitions/index.mdx

```diff
@@ -210,3 +210,7 @@ This is a Release admonition
 :::security[Security]
 This is a Security admonition
 :::
+
+:::docu[Docu]
+This is a Docu admonition it's so beautiful 
```

Comment:
```suggestion
This is a Docu admonition
```

We want to keep the example text in the same format for all - makes it easier to update or translate later.